components/Themes/tests/unit/MDCTonalPaletteTests.m

```diff
@@ -60,6 +60,17 @@ - (void)testTonalPaletteEncodingDecoding {
   XCTAssertEqual(decodedTonalPalette.mainColorIndex, tonalPalette.mainColorIndex);
   XCTAssertEqual(decodedTonalPalette.darkColorIndex, tonalPalette.darkColorIndex);
   XCTAssertEqual(decodedTonalPalette.lightColorIndex, tonalPalette.lightColorIndex);
+
+  // These tests are commented out because of precision bug on apple side when archiving.
+  // Issue #2931.
+  // TODO: Uncomment these tests when this issue is fixed.
+//  XCTAssertTrue([decodedTonalPalette.colors isEqualToArray:tonalPalette.colors]);
```

Comment:
Does this fail with XCTAssertEqualObjects?  Another solution is to switch to single-precision color values in the initializers.

Reply:
Done.